.github/workflows/remove-wheels.yml

```diff
@@ -51,9 +51,11 @@ jobs:
 
       - name: Query package index for packages
         run: |
+          curl https://raw.githubusercontent.com/scientific-python/upload-nightly-action/main/packages-ignore-from-cleanup.txt --output packages-ignore-from-cleanup.txt
```

Comment:
If the checkout action is used instead then you can test versions on branches prior to merge. So can just throw

https://github.com/scientific-python/upload-nightly-action/blob/2b318b62a3436c8e05b7ed8d169a6feff8e9b537/.github/workflows/ci.yml#L24

at the top of `steps` (or anywhere in `steps` before this).

Reply:
This was deliberate, given this is the only file needed from the repo, I opted for the curl rather than using an action that potentially need to be version updated, etc.

Reply:
>  opted for the curl rather than using an action that potentially need to be version updated, etc.

Can you explain more? This doesn't make sense to me. The actions for this workflow are already maintained automatically by Dependabot by `.github/dependabot.yml`. The clone will be trivially fast.

Reply:
dependabot generating the noise would still be noise 🤷‍♀️ 

And the remove logic is not tested anywhere, so I still don't think it's necessary to do the clone at all, and would otherwise just take this file from main.